outputs.tf

```
@@ -36,3 +36,8 @@ output "stage_arn" {
   description = "The ARN of the gateway stage"
   value       = module.this.enabled ? aws_api_gateway_stage.this[0].arn : null
 }
+
+output "log_group_name" {
+  description = "The ARN of the Cloudwatch log group"
+  value       = local.log_group_name
```

```suggestion
  value       = module.this.enabled ? module.cloudwatch_log_group.log_group_name : null
```

Any thoughts on using the module output rather than the local variable?

---

Yeah, I considered that. It doesn't make much sense to have a local that's used only once. I'll change it. 

It should be a bit different since the module won't create a log group if `var.logging_level = "OFF"`.

I think it should be 
`value = local.create_log_group ? module.cloudwatch_log_group.log_group_name : null`

---

That looks good! I was just thinking of keeping with the already established pattern. Thanks for this contribution